test/images/jessie-dnsutils/Dockerfile

```diff
@@ -16,6 +16,11 @@ FROM BASEIMAGE
 
 CROSS_BUILD_COPY qemu-QEMUARCH-static /usr/bin/
 
+COPY fixup-apt-list.sh /
+RUN ["bash", "/fixup-apt-list.sh"]
+
+RUN cat /etc/apt/sources.list
```

Comment:
I'd do this in the previous RUN command or not at all

Reply:
Done